src/main/java/data/Account.java

```diff
@@ -111,4 +114,21 @@ public String getAggregationMode() {
   public void setAggregationMode(String aggregationMode) {
     this.aggregationMode = aggregationMode;
   }
+
+  /** Return a String representation of Account fields. */
+  public String toString() {
+    StringBuilder accountBuilder = new StringBuilder();
```

Comment:
Is this function meant to build a JSON string? Or just a readable serialized format?

If it's meant to build JSON, it probably shouldn't overwrite the toString function.
In its current state, I don't think it produces valid JSON since the keys are missing quotations

Reply:
Either way, I would switch this to using String.format instead of a StringBuilder. StringBuilder adds a lot of awkwardness and verbosity here that would be removed by using String.format

Reply:
Will do.